README.md

```diff
@@ -16,4 +16,24 @@
 
 ## 配置
 
-* `ycy.reminderViewIntervalInMinutes`: 展示提醒页面的时间间隔（分钟）。 (默认值为**60**)
+* `ycy.reminderViewIntervalInMinutes`: 展示提醒页面的时间间隔（分钟）。(默认值为**60**)
+* `ycy.title`: 提示文字。 (默认值为**小哥哥，小哥哥，代码写久了，该休息啦~**)
+
+* `ycy.type`: 类型：defalut:默认图；url：图片地址。(默认值为**defalut**)
```

Comment:
Is there a specific reason to use "defa**lu**t" rather than "defa**ul**t", or is it a typo?